examples/get-ts/index.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright 2016-2021, Pulumi Corporation.
+
+import * as pulumi from "@pulumi/pulumi";
+import * as aws from "@pulumi/aws-native";
+
+const logGroup = new aws.logs.LogGroup("log-test", {
+  retentionInDays: 90,
+});
+
+const fetchedLogGroup = aws.logs.getLogGroupOutput({
+  logGroupName: logGroup.logGroupName.apply((n) => n ?? ""),
```

Comment:
Since you are using the fnOutput variant of the getter, do you need the apply here?

Reply:
It's because it can be null - and in Typescript that's a compilation failure -> `Output<string | undefined> cannot be assigned to Input<string>`